src/sedml/SedBase.cpp

```diff
@@ -1678,7 +1708,16 @@ SedBase::unsetId ()
 
 int SedBase::unsetName()
 {
-  return LIBSEDML_OPERATION_FAILED;
+    mName.erase();
```

Comment:
i dont see how this could ever fail

Reply:
Neither do I, but you see it in a lot of libsbml code, and it's generated by Deviser, where it landed in some other function I copied and pasted into here.  I'll go ahead and remove the fail condition; I also don't think it's possible to reach it.